apps/labs/posts/spot-the-differences.md

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Spot the differences: what is new in Spyder 5?'
+published: April 16, 2021
+author: isabela-presedo-floyd, juanita-gomez
+description: 'In case you missed it, Spyder 5 was released at the beginning of April! This blog post is a conversation attempting to document the long and complex process of improving Spyder's UI with this release.'
+category: [IDEs]
+featuredImage:
+  src: ../public/posts/spot-the-differences/feature.png
+  alt: 'A comparison of the user interfaces of Spyder 5 versus Spyder 4. Spyder 5 has a more steamlined
+  interface.'
+hero:
+  imageSrc: ../public/posts/spot-the-differences/hero.png
+  imageAlt: 'A comparison of the user interfaces of Spyder 5 versus Spyder 4. Spyder 5 has a more steamlined interface.'
+---
+
+![Spyder 5 versus Spyder 4](../public/posts/spot-the-differences/spyder5-header.png)
```

Comment:
```suggestion
```
This 'header image' was used as the 'hero image' for the new website. Since the hero image is displayed on top of the page by default, should I delete the 'header image' in the post?